unitex/src/fr/umlv/unitex/frames/GraphPathDialog.java

```diff
@@ -237,6 +266,51 @@ private JPanel constructGraphNamePanel() {
 		panel.add(graphName, BorderLayout.CENTER);
 		return panel;
 	}
+	
+	/**
+	 * Constructs panel with list file output location and browse button to allow user to select other file/location
+	 * issue #61
+	 * @return
+	 */
+	private JPanel constructListFileNamePanel() {
+		
+		outputFileName.setText(ConfigManager.getManager()
+				.getCurrentLanguageDir() + String.valueOf(File.separatorChar) + "list.txt");
+		final JPanel panel = new JPanel(new BorderLayout());
+		panel.add(new JLabel(" Output file: "), BorderLayout.WEST);
```

Comment:
This is a hacky solution here. JLabel.setPreferredSize() didn't work as expected. I didn't want to change to FlowLayout without your approval.